apps/juicebox_web/assets/js/reactions/styles.scss

```diff
@@ -0,0 +1,21 @@
+.reactions {
+  width: 100%;
```

Comment:
Properties should be ordered display, width

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)